extras/asr-services/common/base_service.py

```
@@ -194,7 +194,7 @@ async def transcribe(file: UploadFile = File(...)):
         except Exception as e:
             error_time = time.time() - request_start
             logger.exception(f"Error after {error_time:.3f}s: {e}")
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)